packages/rocketchat-lib/client/models/FullUser.js

```diff
@@ -0,0 +1,7 @@
+
+RocketChat.models.FullUser = new class extends RocketChat.models._Base {
```

Comment:
Why this separation, I didn't understand the idea.

Reply:
the issue is since I'm updating/inserting records on `users` collection on the client by hand, Meteor will throw out error if I subscribe to a publication that would add records to `users` collection.

so I had to change the [fullUserData](https://github.com/RocketChat/Rocket.Chat/pull/12800/files#diff-ea4ac869170343d111ac3c90a4c5b92c) publication to publish on this new virtual collection.